utils/parametric/_library_client.py

```diff
@@ -117,6 +117,21 @@ def span_set_resource(self, span_id: int, resource: str) -> None:
     def span_set_meta(self, span_id: int, key: str, value) -> None:
         raise NotImplementedError
 
+    def span_set_baggage(self, span_id: int, key: str, value: str) -> None:
```

Comment:
Why is there a `span_id` while baggage API is independent of span / tracing API?